build.gradle

```diff
@@ -13,12 +13,10 @@ allprojects {
         implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: jacksonVersion
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: apacheCommon
 
-        //Dependency to converter Duration to string and vice versa
+        //Dependency to convert Duration to string and vice versa, CreationTimestamp and UpdateTimestamp annotations

```

Comment:
tried to remove this dependency but I need to use these imports: import org.hibernate.annotations.CreationTimestamp; import org.hibernate.annotations.UpdateTimestamp;
for "create_date" and "last_update_date" to work properly